pkg/kubernetes/upgrade.go

```diff
@@ -35,15 +35,13 @@ var crds = []string{
 	"configuration",
 	"subscription",
 	"resiliency",
-	"pluggable-components",
```

Comment:
I may be missing the background, but this is removed intentionally?

Reply:
Ok, got it, just read discord's chat.

Reply:
It's also here FYI https://github.com/dapr/dapr/issues/5210#issuecomment-1258479834